Sources/ArcGISToolkit/Documentation.docc/AugmentedReality.md

```diff
@@ -10,17 +10,17 @@ reality experiences for common AR patterns including Flyover, TableTop, and Worl
 `FlyoverSceneView` provides an augmented reality flyover experience that allows you to 
 explore a scene using your device as a window into the virtual world. The experience begins
 with the ArcGIS Scene's camera positioned over an area of interest. Walk around and reorient
-the device to focus on specific content in the scene. To learn more about using the ``FlyoverSceneView`` see the <doc:FlyoverSceneViewTutorial>.
+the device to focus on specific content in the scene. To learn more see the <doc:FlyoverSceneViewTutorial>.
```

Comment:
I'm not sure that we should make this change because these symbol links are helpful on other platforms. We host the iOS docC archive, not the mac catalyst docC archive so these warnings won't be there. I remember reading through this AR doc and I found these links helpful. Thoughts?

Reply:
I had suggested to @des12437 that she remove them because component name is already provided via the paragraph headers.

Also it looks like these links aren't live on the current published web version so I'm not sure what the targeted platform is when the docs are built for hosting:
<img width="1504" alt="Screenshot 2024-11-14 at 2 30 28 PM" src="https://github.com/user-attachments/assets/a1f11091-6d6d-43ae-8cc1-bc615c162f15">

Hopefully DocC supports compiler directives of some sort eventually.

@mhdostal thoughts?

Reply:
>  I can double-check this because now I am curious.

Yeah looks like it is using the iOS DocC archive, hmm weird.

Reply:
Oh, I know why. The symbol links were added in 200.6 and the doc you are looking at is for 200.5.

Reply:
Yup just saw that too. I'm fine with reverting - just wish there was a way to eliminate all possible warnings.

Reply:
>  just wish there was a way to eliminate all possible warnings.

I agree with that :face_exhaling:

Reply:
I can revert these changes for now.

Reply:
> Hopefully DocC supports compiler directives of some sort eventually.

Maybe we can log an issue like https://github.com/swiftlang/swift-docc/issues/255 but specify our need to allow skipping certain symbols for visionOS.

Reply:
> Maybe we can log an issue like https://github.com/swiftlang/swift-docc/issues/255 but specify our need to allow skipping certain symbols for visionOS.

Good idea @yo1995. I can log an issue.

Reply:
https://github.com/swiftlang/swift-docc/issues/1098